### ISSUE TITLE
Greek accented letters are missing inside \emph{} or \textbf{} commands

### ISSUE THREAD

**Issue by ralessi** [skosch/Crimson#48]
Thank you very much for providing such a beautiful font!

I noticed that Greek accented letters disappear inside `\emph{}` or `\textbf{}`.  Try this for example:

````
\documentclass{article}
\usepackage{fontspec}
\usepackage{crimson}

\begin{document}
λέγων \emph{λέγων} \textbf{λέγων}
\end{document}
````

**Comment by katef** [comment]
Roman on the left, Bold on the right. What on earth happened to that accent!

![g](https://cloud.githubusercontent.com/assets/1371085/12998061/22fb5204-d137-11e5-8612-d0e514708a79.png)

**Comment by ralessi** [comment]
Well, further investingating this issue leads me to think that it may not be related to Crimson.  If I run the above mwe through xelatex, I get this:
![xelatex](https://cloud.githubusercontent.com/assets/389498/13000461/e2eb89da-d15a-11e5-9873-90576d0a72a6.png)
But If I run it through lualatex (which I did in the first place), I get this:
![lualatex](https://cloud.githubusercontent.com/assets/389498/13000507/3f066bae-d15b-11e5-9a07-4628d8e5de87.png)

Having said that, don't you find the acute above ε inside \textbf{} a bit forward?

**Comment by katef** [comment]
Yes, the accent is more than a bit forward! That's what I was posting my image to show.

My guess is that LuaLaTeX sees that the accent is entirely outside the glyph's bounding box, considered it an error, and perhaps opts to not render the glyph accordingly. That's just a guess, but this is definitely a bug in Crimson.

**Comment by ralessi** [comment]
Ok, I misunderstood what you meant, although it was very clear. So yes, what on earth happened to that accent,to say nothing about the one not showing up at all under lualatex!
